sdk/agrifood/azure-agrifood-farming/samples/README.md

```diff
@@ -0,0 +1,104 @@
+products:
+  - azure
+  - azure-farmbeats
+urlFragment: farmbeats-samples
+---
+
+# Azure FarmBeats samples for Python client
+
+These sample programs demonstrate some common use case scenairos for the [FarmBeats Python client library][python_sdk].
+
+- [sample_hello_world.py][hello_world_sample] demonstrates the most basic operation that can be performed - creation of a Farmer. Use this to understand how to create the client object, how to authenticate it, and make sure your client is set up correctly to call into your FarmBeats endpoint.
+
+- [sample_farm_hierarchy.py][farm_hierarchy_sample] demonstrates how to create a simple [farm hierarchy][farm_hierarchy_docs], comprising of Farmer, Farm, Field and Boundary.
+
+- [sample_farm_hierarchy_complete.py][farm_hierarchy_complete_sample] builds on the previous sample to add to the previous sample, and demonstrate the relationships between Crops, Crop Varities, Seasons and Seasonal Fields and the other farm hierarchy objects.
+
+- [sample_satellie_download.py][satellie_download_sample] demonstrates FarmBeats' satellite integrations and how to ingest satellite imagery into the platform, and then download them onto a local directory. This demonstrates key concepts of how to create a job in FarmBeats, and how to poll on the completion of the job using the SDK. In the [corresponding async sample][satellie_download_async_sample] we demonstrate how to handle concurrent download of potentially hundreds of files by limiting the max concurrency using semaphores.
+
+- [sample_attachments.py][attachments_sample] demonstrates FarmBeats' capabaility of storing arbitrary files in context to the various farm hierarchy objects. We first attach some files onto a farmer and a farm, and then download all existing attachments for the farmer onto a local directory.
+
+- [sample_cascade_delete.py][cascade_delete_sample] demonstrates the usage of cascade delete jobs to perform cleanup of farm hierarchy objects. A cascade delete job handles the recursive deletion of all dependent data for the given parent resource. Use this to clean up the sample resources that you create in the other samples.
+
+
+Additionally, for each sample, there are corresponding files in the [`samples/async`][async_samples] directory
+
+## Prerequisites
+
+To run the samples, you need:
+
+- A [python][get_python] environment. Supported versions are 2.7 and 3.6+.
+- An Azure subscription. Create a free subscription [here][azure_free_sub].
+- A FarmBeats resource. See [installation docs][install_farmbeats] to create a new FarmBeats resource.
+
+Additionally, there are some specific prerequisites if you want to leverage third party integrations into FarmBeats:
+
+- A subscription to a [supported weather provider][weather_docs], to run the weather sample.
+
+## How to run the samples
+
+### Install the dependencies
+
+To run the samples, you need the following three dependencies.
+```bash
+pip install azure-agrifood-farming azure-identity aiohttp
```

Comment:
we shouldn't force users to use aiohttp, so just remove aiohttp from this pip install link

Reply:
Without aiohttp, we get an error that says aiohttp was not found.

Reply:
that's by design. Basically, you can use another async library if you wish to (i.e., you can use requests with asyncio). However, if you don't have any async http library, we tell you to install `aiohttp`. So basically we want to keep this behavior, where we don't force people to use `aiohttp`, but if you don't have any libraries installed, we suggest aiohttp as the one to install